recipes-core/playready-headers/playready-headers_4.2.bb

```diff
@@ -0,0 +1,26 @@
+SUMMARY = "Playready headers"
+LICENSE = "CLOSED"
```

Comment:
This is an opensourced repo so it is unusual to refer to a closed license component.  In particular, are some filenames and paths from a non-open zipfile being made visible?

Reply:
Removed the playready-headers_4.2 recipe. Updated the readme to point to another location for it.